CMakeLists.txt

```diff
@@ -1782,8 +1782,20 @@ target_sources(${PROJECT_NAME} PRIVATE
         core/gdxsv/gdxsv_patch.inc
         core/gdxsv/gdxsv_emu_hooks.cpp
         core/gdxsv/gdxsv_emu_hooks.h
+		core/gdxsv/gdxsv_translation.h
+		core/gdxsv/gdxsv_translation.cpp
+		core/gdxsv/gdxsv_translation_patch.inc
+		core/gdxsv/gdxsv_CustomTexture.h
+		core/gdxsv/gdxsv_CustomTexture.cpp
```

Comment:
rename `gdxsv_CustomTexture` to `gdxsv_custom_texture`

Reply:
The special camelCase here is to align with 
`core/rend/CustomTexture.cpp` and `core/rend/CustomTexture.h`

So would be easier to navigate with its parent class file
<img width="566" alt="image" src="https://user-images.githubusercontent.com/602245/190469246-9ed33e87-22e1-4897-a6a3-553fc1992b02.png">

Reply:
Mixed snakecase and camelcase names are very strange.
I had a little thought, let's use snake_case to avoid confusion when we faced with similar cases in the future.